CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## [2.0.120230817]
+- Updating to latest B2k CLI version majorly for remediating security vulnerability for CBL mariner. Please refer for release notes [here](https://github.com/Azure/Bridge-To-Kubernetes/blob/main/CHANGELOG.md#10202308112)
```

Comment:
💡 Idea/suggestion/thoughts: Changelog could be abstract but it is one place which is easily the source of truth so if you want we can verbatim as bullet point put things we have directed et. al. so that we don't have to see what commits are made between releases. What do you think? I am good with whatever we decide. 

Rest looks good, thank you so much for this. ❤️❤️🙏

<img width="1615" alt="Screenshot 2023-08-18 at 2 49 43 AM" src="https://github.com/Azure/vscode-bridge-to-kubernetes/assets/6233295/ee59ce34-cc8a-4be7-bcd1-e6cf0f0899cf">

Reply:
which is standing true because this changelog is accurate from last release of July. Thank you.

So moving forward we will keep changelog true to commit history, so one place to read all information.